docs/breaking.md

```diff
@@ -1,8 +1,17 @@
 # Breaking Changes
 
+## Nov 21, 2019
+
+The endpoint to [Get a pipeline's workflows](https://circleci.com/docs/api/v2/#get-a-pipeline-39-s-workflows) `GET /pipeline/{pipeline-id}/workflow` has been released. 
+
+The `workflow` key in the [Get a pipeline](https://circleci.com/docs/api/v2/#get-a-pipeline) `GET /pipeline/{pipeline-id}` response is now deprecated and is scheduled to be deleted on Dec 9, 2019
+
+Instances where the `workflow` key is being referenced from the [Get a pipeline](https://circleci.com/docs/api/v2/#get-a-pipeline) `GET /pipeline/{pipeline-id}` response should be modified to make a call to [Get a pipeline's workflows](https://circleci.com/docs/api/v2/#get-a-pipeline-39-s-workflows) `GET /pipeline/{pipeline-id}/workflow` instead. 
+
+
 ## Nov 11, 2019
 
-- The `/workflow/:workflow-id/jobs` endpoint has been moved to `/workflow/:workflow-id/job` for consistency. The old route is still working at this point in time, but deprecated and will be removed on November 18th, 2019.
+The `/workflow/:workflow-id/jobs` endpoint has been moved to `/workflow/:workflow-id/job` for consistency. The old route is still working at this point in time, but deprecated and will be removed on November 18th, 2019.
```

Comment:
```suggestion
The `/workflow/:workflow-id/jobs` endpoint has been moved to `/workflow/:workflow-id/job` for consistency. The old route was removed on November 20th, 2019.
```

Reply:
merged to quickly but manually patched in your suggested change